Tools/autotest/arducopter.py

```diff
@@ -6548,6 +6552,8 @@ def fly_rangefinder_drivers(self):
                     self.set_parameter("RNGFND%u_TYPE" % (offs+1), rngfnd_param_value)
             self.customise_SITL_commandline(command_line_args)
             self.fly_rangefinder_drivers_fly([x[0] for x in do_drivers])
+            for n in range(3):
+                self.set_parameter("RNGFND%u_TYPE" % (n+1), 0)
```

Comment:
This should instead be done with a `context_push()` and `context_pop()` around the `for` loop.  That way we revert all the relevant state, including the serial protocol numbers.

Reply:
Fixed it . Thanks.